src/main/java/appeng/block/qnb/QuantumRingBlock.java

```diff
@@ -32,8 +32,8 @@
 public class QuantumRingBlock extends QuantumBaseBlock {
 
     private static final VoxelShape SHAPE = createShape(2.0 / 16.0);
-    private static final VoxelShape SHAPE_CORNER = createShape(4.0 / 16.0);
-    private static final VoxelShape SHAPE_FORMED = createShape(1.0 / 16.0);
+    private static final VoxelShape SHAPE_CORNER = createShape(2.0 / 16.0);
+    private static final VoxelShape SHAPE_FORMED = createShape(0.0 / 16.0);
```

Comment:
I think it would be a bit nicer to try to match the exact shape of the block. :)

Reply:
Are you specifically referring to the middle blocks where there is a slight offset extruding out from each face, to make the hitbox match the extrusion?

Another side not, I noticed something odd in the QnbFormedBakedModel.class as shown below. The constants for the center and corner blocks are unused. I was wondering if they were intended to be used, because if implemented with those values it makes the corner blocks smaller, making the structure look much more like a ring. The constants from lines 71 and 72 are the only ones used in the whole file, the others are never referenced.

https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8f542e5047f120e8f27c861ec6400408f2cbc1e4/src/main/java/appeng/block/qnb/QnbFormedBakedModel.java#L71-L78

Reply:
> Are you specifically referring to the middle blocks where there is a slight offset extruding out from each face, to make the hitbox adjust at the extrusion?

Yes

> Another side note, I noticed something odd in the QnbFormedBakedModel.class as shown below. The constants for the center and corner blocks are unused. I was wondering if they were intended to be used, because if implemented with those values it makes the corner blocks smaller, making the structure look much more like a ring. The constants from lines 71 and 72 are the only ones used in the whole file, the others are never referenced.

I will check, but I think it might have been from before the 2020 texture update.

Reply:
Fixed the hitbox in recent commit. Now properly matches the extrusions on the model. Tested it by walking on top of the formed blocks and standing on the side extrusion of a formed block.